src/RemoteInboxNotifications/DataSourcePoller.php

```diff
@@ -51,17 +51,28 @@ private static function read_data_source( $url ) {
 		$response = wp_remote_get( $url );
 
 		if ( is_wp_error( $response ) || ! isset( $response['body'] ) ) {
+			// phpcs:ignore
+			error_log( 'Error getting remote inbox notification data feed from ' . $url );
```

Comment:
@becdetat what do you think about using Woo's built-in logger for this instead: https://github.com/woocommerce/woocommerce/blob/a5e956e7bf2e86e25568a6cfac6b68a52ff4a9a5/includes/class-wc-background-updater.php#L39

Reply:
Gosh, OK :-)